examples/config-dev.yaml

```diff
@@ -17,6 +17,8 @@ staticClients:
 - id: example-app
   redirectURIs:
   - 'http://127.0.0.1:5555/callback'
+  - 'http://localhost:9000/auth/callback'
+  - 'urn:ietf:wg:oauth:2.0:oob'
```

Comment:
If I don't add that, I get `invalid_request: Unregistered redirect_uri ("urn:ietf:wg:oauth:2.0:oob").` when trying certain features.

But you're right, this shouldn't be in this repo.